Substrate.NetApi.TestNode/ExtrinsicsTest.cs

```diff
@@ -73,11 +74,14 @@ public async Task Extrinsic_SubmitAndWatchExtrinsicAsync()
         {
             var iType = new BaseVec<U8>(new U8[] { (U8)0x04, (U8)0xFF });
 
+            var assetCharge = ChargeAssetTxPayment.NewWithAsset(0, new I32(0));
```

Comment:
This requires that we have a liquidity pool on the chain with the asset with id 0 and our native token. On paseo we have that, and this test was run against a paseo node.